lib/portlayer/network/network.go

```diff
@@ -94,8 +74,13 @@ func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSou
 		}
 
 		var netctx *Context
-		netctx, err = NewContext(*bridgeRange, *bridgeWidth, &config)
+		netctx, err = NewContext(ctx, sess, *bridgeRange, *bridgeWidth, &config)
 		if err != nil {
+			log.Warn(err)
```

Comment:
Its _supposed_ to return err (line 120), but I think right now it always return `nil`. I have a fix for this in vch restart changes.